pkg/proxy/iptables/proxier.go

```diff
@@ -150,8 +149,8 @@ type serviceInfo struct {
 
 // internal struct for endpoints information
 type endpointsInfo struct {
-	ip            string
-	localEndpoint bool
+	endpoint string // TODO: should be an endpointString type
+	isLocal  bool
```

Comment:
While you're at it, can you:

func (e *endpointsInfo) String() string {
	return fmt.Sprintf("%v", *e)
}

So that something reasonable can be printed in testcases?